build.gradle

```diff
@@ -54,7 +54,7 @@ def javaCodeCheckedProjects = subprojects.findAll { !['util', 'mongo-java-driver
 configure(coreProjects) {
     evaluationDependsOn(':util')
     group = 'org.mongodb'
-    version = '3.11.0-SNAPSHOT'
+    version = '3.11.0-mongot'
```

Comment:
I'm not sure how many dashes are allowed in Maven versions, but you probably want something more like:

     3.11.0-mongot-SNAPSHOT

Reply:
Seems reasonable, the second dash seems OK as part of the version qualifier per https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges and . It did not give our build issues.